github/github.go

```diff
@@ -654,10 +654,16 @@ func (config *GithubConfig) ForEachIssueDo(labels []string, fn PRFunction) error
 
 		pr, _, err := config.client.PullRequests.Get(config.Org, config.Project, *issue.Number)
 		if err != nil {
-			glog.Errorf("Error getting pull request: %v", err)
+			glog.Errorf("Error getting pull request %v: %v", *issue.Number, err)
 			continue
 		}
-		if err := fn(pr, issue); err != nil {
+		// re-get the issue, so it won't be stale.
+		currentIssue, _, err := config.client.Issues.Get(config.Org, config.Project, *issue.Number)
+		if err != nil {
+			glog.Errorf("Error getting issue %v: %v", *issue.Number, err)
+			continue
+		}
+		if err := fn(pr, currentIssue); err != nil {
```

Comment:
Hmmmm, how about we change "fetchAllIssuesWithLables()" to return an []int. We then `Get` each issue at the top of this function and should "recheck" the "open" state here. I'd rather we didn't have an outdate `github.Issue` to confuse us down the road...